### PR TITLE
Big oh lord update, actually read this community

### PR DESCRIPTION
**Pull request by scarwolff**
## Description
dear community, read and discuss this.

## Changelog (necessary)
:cl:
add: Tweaked crafting menu to be more organised.
add: Added banker and barkeep job roles all with loadouts.
add: Made autolathe into workshop, requires skill to use now, added to all relevant roles and boo in WMV. - REMOVED TEMP
add: Renamed everything bluespace to quantum.
add: Renamed everything enzyme to yeast.
add: Tweaked plasma to graphite on computers.
add: Added fully functional meat grill, only grill once, lest you burn your meat son.
add: Commented out powergame traits (big league, iron fist, RDS). - REMOVED TEMP
add: Tools are now in 4 tiers, along with speed differences and crafting difficulty- can build crude and basic tools in crafting, others require RnD or Workshop.
add: Sleeper now requires being a doctor, no more clinic tgheads.
add: City gate now starts open.
add: Backpack resprite.
add: like 40 new fallout drinks and cocktails, sorted boozeomat and new BOOK, with all drinks recipes in game.
add: Booth table now works as a table.
add: bar is more retro kitchen and has shutters and bartender only access, same with bank.
add: Banker starts with a shitton of caps.
add: Tweaked settler roles to make it less hiveminded and more of a society.
add: More CASP packages.
add: All wasteland plants now grind into tea, all with effects and stuff, mutjuice etc, used in recipes, check the book
add: Being thrown into a wall now knocks you down.
add: Curbstomping on every body part, groinstomp, curbstomp and bone snap when standing on your enemy vertically and shift-middleclick them to initiate stomp. - REMOVED TEMP
add: Resprited a ton of shit.
add: Unburied some cool sprites from fallout folder.
add: Showers and other stuff (GRILL!!) now craftable.
add: Can craft cool cowboy hats, with actual inhand non-codersprites now.
tweak: Tweaked all the maps to fully remove the forbidden dispensers, added more lootable underground ones, now rare, added more vending machines and lootracks to map.
add: Probably more stuff.
del: My Soul (help).
del: Purrbation.
del: Sunnyvale tghead things, Gene manip etc.
soundadd: added bone snap, wall throw and the infamous grill sounds.
/:cl:

### CODE REVIEW

**Comment by AsterixCodix** [comment]
What is that "Probably more stuff" addition? Care to elaborate?
So, the fact that the community voted against making the Workshop require a skill to use, is being completely ignored here.
Why not reduce the points gain from RDS instead of removing it?

**Comment by Mikierpg** [comment]
RDS shouldnt be removed, we have a number of people who actually rp with it. just make it a neutral trait. 
The Crafting menu option "salvage for rods" and "Salvage for plating" require 0 ingredients, allowing you to infinitely make metal. 
Why was the stim recipe changed? it was fine.
The granter gives the technophreak trait? Is that not the one that allows you to use RND consoles?
Why were lathes gatelocked despite there being no approval for that major of a change?

**Comment by Laus4Deus** [comment]
As I told you in private messages, upload a PR that has the changes that I approved, without the ones that I have told you for three days to take away.

@judgex close this pr, please.

**Comment by scarwolff** [comment]
dont close it man, im gonna make the changes on the PR when they come

**Comment by Laus4Deus** [comment]
Oh sure, I want to see you following the document I sent you.
Begin.

**Comment by zuliouse** [comment]
_"add: Made autolathe into workshop, requires skill to use now, added to all relevant roles and boo in WMV."_

Wasn't this voted against by the community?

_"add: Probably more stuff."_

Can you clarify what this means?

**Comment by annoyedbunny** [comment]
im cool with everything sept removal of purrbation, keep that in please we need our admin spawned catgirls

**Comment by Girdo** [comment]
Alrighty. A lot of the changes are minor quality of life. I'll comment on things in the changelog I particularly enjoy or disagree with. I see the minor things exist, I appreciate them, but I'm not gonna repeat "this is a good change, quality of life" on every tweak to the UI or item names.

- **Added banker and barkeep job roles all with loadouts.** Haven't gotten to experiment much with barkeep. Got to experiment with banker for a solid thirty seconds. From what I saw of the Banker loadouts, bank itself, etc., it's absolutely stellar. It's a cool building, you get cool clothes and gear, it's an excellent idea, didn't encounter any bugs. The one tiny change I'd suggest because I'm petulant is that the Classy loadout has significantly inferior weaponry (the others get an Uzi and revolver, Classy gets a regular cane and a zipgun). If it had a cane-sword (which I believe is already in TG code, don't quote me) or similarly robust classy weapon, it'd be the pinnacle of perfection and I'd leave the Legion to main Banker.
 
- **Made autolathe into workshop, requires skill to use now, added to all relevant roles and boo in WMV.** This caused a shitstorm, so I'll be brief. I personally don't have a problem with this change despite the majority being against it. Autolathes don't even give you that much useful gear. Like, maybe flashlights, but you can get them en masse from vending machines. That being said, in a case where a lot of people voted for this change to not exist, it might be best to leave it out of the final update.

- **Commented out powergame traits (big league, iron fist, RDS).** The second- and I think last- questionable decision. As the game currently stands, traits are pretty much useless. I take Poor Aim on a regular basis because I'm a Legion main and guns are stupid, and take no beneficial traits because there's nothing really worth it. Big Leagues, Iron Fist, etc. are pretty minimal in actual buffs. Do they give an advantage? Yeah, technically. But _barely_. If we want traits to not affect anything mechanically, we should just not have traits at all since no one really cares if you're hard to get drunk or have to wear glasses. If we do want traits to have mechanical impact, banning the two actually beneficial traits is questionable. You'll just go from people choosing only a few traits to absolutely nobody using traits.

- **Sleeper now requires being a doctor, no more clinic tgheads.** I'm behind this 100%. Complex medicinal machinery is complex, this is not an AutoDoc. Stop hitting random buttons, wastelander.

- **Adds ~40 new fallout drinks and cocktails, sorted boozeomat and new BOOK, with all drinks recipes in game.** I love this. More bartender flavor. The change that makes stuff grindable into tea is just as good.

- **Tweaked settler roles to make it less hiveminded and more of a society.** Fair enough, I can get behind this. Encourages RP and interaction rather than infinite farmers and civilians.

- **Being thrown into a wall now knocks you down.** Minor change, but realistic and adds an interesting twist to combat. Though it's rather niche, very few people actually use throws in combat. Most just firearm you horizontal.

- **Curbstomping on every body part, groinstomp, curbstomp and bone snap when standing on your enemy vertically and shift-middleclick them to initiate stomp.** Eh, I'm questionable on this one as well. Even as a lover of melee/unarmed weapons as a Legionary, seems a bit gratuitous. I'm not opposed to it, to be clear. Just not super excited about the addition.

Overall, I don't actually have any _problems_ with this. Even the most punishing of things just gets a "meh, not gonna explode fireworks but okay". And there are definitely some gems in here.

Oh, almost forgot!

- **Removes Purrbation.** Thank you. I no longer have to murder staff members when they toggle it for shits and giggles.

**Comment by pilotbland** [comment]
please don't fucking remove purrbation.

**Comment by scarwolff** [comment]
GAMERS LISTEN UP, THE FOLLOWING HAS BEEN COMMENTED OUT TEMPORARILY:
AUTOLATHE CHANGE (yall gonna shit yourself when we rebase, man o man)
CURBSTOMPING
THE REMOVAL OF POWERGAME TRAITS?
STIMPACK RECIPE (all doc mains listen up, you gotta abuse this more, put it in the light)
IF THAT IS ALL, then this should be ready to merge :)
IF NOT, say something idk, communicate you chimps.

**Comment by Girdo** [comment]
> The Crafting menu option "salvage for rods" and "Salvage for plating" require 0 ingredients, allowing you to infinitely make metal.

Was this bug fixed?

**Comment by Laus4Deus** [comment]
@Girdo That's the point, which I kinda liked. It takes two minutes and you can find a sheet of metal or a rod. Hmm, but I doubt someone would bother for a rod.

**Comment by Laus4Deus** [comment]
Hm, I need the `/mob/proc/ShiftMiddleClickOn(atom/A)
	src.pointed(A)
	return` to be back

File: code/datums/datumvars.dm [hunk 1]
```diff
@@ -1241,38 +1241,6 @@
...
-				to_chat(usr, "This can only be done to the basic human species at the moment.")
-				return
-
-			if(!H)
-				to_chat(usr, "Mob doesn't exist anymore")
-				return
-
-			var/success = purrbation_toggle(H)
-			if(success)
-				to_chat(usr, "Put [H] on purrbation.")
```
[the remark this answers is not in the document]
Comment (scarwolff) [reply code/datums/datumvars.dm:1263]: i aint fixing your personal qualms

**Comment by scarwolff** [comment]
there we go all fixed, pointing back, and curbstomping needs some adjustment, finally, dare i say, ready to merge

File: code/datums/datumvars.dm (hunk 1 above)
[the remark this answers is not in the document]
Comment (scarwolff) [reply code/datums/datumvars.dm:1263]: making this a personal issue jando, not cool

**Comment by scarwolff** [comment]
aight fixed, truly ready to merge, mouse pointing still an issue but thats gonna have to be done on a seperate PR. (because github)

**Comment by Girdo** [comment]
> @Girdo That's the point, which I kinda liked. It takes two minutes and you can find a sheet of metal or a rod. Hmm, but I doubt someone would bother for a rod.

Ah. I failed to notice the timer. That's perfectly reasonable then. For whatever reason I read it as 12 seconds instead of 120

**Pull request merged**